agent/lib/pbench/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+import sys
+
+import configtools
+
+opts, _ = configtools.parse_args()
```

Comment:
Can we move this to module function like we discussed before?  Having this setup when the module is imported makes it difficult to unit test effectively, and is not needed right when the module is loaded.